src/main/java/com/groupone/lbls/model/Book.java

```diff
@@ -11,18 +11,15 @@
     private int quantity;
     private String keywords;
 
-	public Book(int id, String ISBN, String title, String author, String genre, String publisher, String publisherYear,
```

Comment:
I think this one makes more sense because we have the publish_year column in DB.